apollo-federation.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'graphql', '>= 1.9.8'
 
-  spec.add_runtime_dependency 'google-protobuf', '~> 3.7'
+  spec.add_runtime_dependency 'google-protobuf', '~> 3.13.0'
```

Comment:
Wanted to only keep the minor to `3.13` since this PR is only about removing the platform lock. 

Also guessing typically if we wanted to lock versions, having it locked to patch and having the maintainers manually bump minor/major versions is more intention revealing.